pipeline/reference.py

```diff
@@ -0,0 +1,87 @@
+'''
+Schema of subject information.
+'''
+import datajoint as dj
+
+schema = dj.schema('gao2018_reference')
+
+@schema
+class BrainLocation(dj.Lookup):
+    definition = """
+    brain_location_name: varchar(64)
+    ---
+    brain_location_full_name: varchar(128)
+    """
+    contents = [
+        ['Fastigial', 'Cerebellar fastigial nucleus'],
+        ['ALM', 'Anteriror lateral motor cortex']
+    ]
+
+@schema
+class Hemisphere(dj.Lookup):
+    definition = """
+    hemisphere_name: varchar(32)
```

Comment:
Let's call the attribute `hemisphere` rather than `hemisphere_name` and make the datatype shorter, e.g. `varchar(8)`